LOOP_RATE: 60.0
WHEEL_BASE_HALF_M: 1.9035
THROTTLE_MAX: 1.0
THROTTLE_MIN: 0.0
P_GAIN: 3.0
I_GAIN: 0.0
D_GAIN: 1.0
TARGET_VELOCITY_MS: 15.0
LOOKAHEAD_DIST_M: 3.0
STANLEY_CONSTANT: 10.
MIN_ARCTAN_NUMERATOR: 3.0
CYBERTRUCK_MAX_STEERING_DEG: 35.0
CYBERTRUCK_MIN_STEERING_DEG: -35.0